elide-model-config/src/main/resources/elideTableSchema.json

```diff
@@ -191,22 +191,7 @@
                             "type": "string"
                         }
                     },
-                    "dependencies": {
-                        "values": {
-                            "not": {
-                                "required": [
-                                    "tableSource"
-                                ]
-                            }
-                        },
-                        "tableSource": {
-                            "not": {
-                                "required": [
-                                    "values"
-                                ]
-                            }
-                        }
-                    }
+                    "eitherTableSourceOrValues": {}
```

Comment:
Why did we do this swap?  Was the error hard to follow?

Reply:
yes.